app/scripts/models/page-refresh-watch.js

```diff
@@ -0,0 +1,66 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * Watch for page refreshes.
+ *
+ * Listen for `show-view` and `show-sub-view` messages and check if the user
+ * has refreshed the page. If so, a `<screen_name>.refresh` event is
+ * logged to metrics.
+ */
+
+define([
+  'backbone',
+  'lib/storage'
+], function (Backbone, Storage) {
+  'use strict';
+
+  function isRefresh (refreshMetrics, screenName) {
+    return refreshMetrics && refreshMetrics.view === screenName;
```

Comment:
...and just to underline that page/screen/view confusion, here `view` is compared to `screenName`.